pptx/table.py

```diff
@@ -384,108 +384,31 @@ def vertical_anchor(self):
     def vertical_anchor(self, mso_anchor_idx):
         self._tc.anchor = mso_anchor_idx
 
-    @property
-    def borL(self):
-        """
-        Get Left Border instance containing all the formatting options.
-        """
-        return self._tc.tcPr.borL
 
-    @lazyproperty
+    @property
```

Comment:
I have two questions here:
1. should we have something like `self.border_left` here so we don't have to initialize `BorderFormat` when dealing with the same cell?
2. I was looking at some of Steve's other code and it seems that he uses the `@lazyproperty` when returning an instance of an object and `@proeprty` when returning a variable from a class. What was your reasoning for changing this to `@property`. This is more for my curiosity.

Reply:
So my understanding of property vs lazy property was not complete but ultimately I believe I was correct.  So looking at the actual definition of the `lazyproperty` decorator, it is used to get a cached property value and then return it.  I don't think we need to be using a cache at all for these properties.

As far as #1, I'm not quite sure what you mean.  Each border is its own object so each object needs to be initialized with the correct oxml object.  If I'm missing what you are saying, can you please clarify?

Reply:
For 1, I meant if we should add an attribute to the _Cell object such that we have in the `__init__` function:

```python
class _Cell(..):
    def __init__(...):
        ....
        self.left_border = None
        self.right_border = None
        .....
```
That way we can just have this inside of the `border_left` function

```python
@property
def border_left(self):
   if not self.border_left:
      self.border_left_line = BorderFormat(self._tc.tcPr, 'left')

  return self.border_left_line
```
So we are not initializing an object every time we call cell.border_left, cell.border_right. Also, this could already be in place and I'm just lacking knowledge of property.

Reply:
Ahh, I see what you are saying.  So I haven't seen that approach being used anywhere else.  This is consistent with how Objects are created in this situation for things like other Lines as well as Fills.  Typically, we aren't updating a line multiple times so this shouldn't be an issue and I think if we did have to initialize the small object multiple times, any performance hit would be negligible.

Reply:
I did a bit more research on the differences between `property` and `lazyproperty`. This article does a good job in my opinion: https://medium.com/better-programming/how-to-create-lazy-attributes-to-improve-performance-in-python-b369fd72e1b6.

You bring up a good point though. If we are modifying the same border multiple time, then we need to re-think the higher-level code.